templates/boltzmann/middleware/honeycomb.ts

```diff
@@ -9,77 +19,180 @@ import beeline from 'honeycomb-beeline'
 import isDev from 'are-we-dev'
 void `{% endif %}`;
 
+let otelTracer: OtelTracer | null = null;
+
+const OTEL_NAMESPACE = 'boltzmann'
+const OTEL_REQ_NAMESPACE = `${OTEL_NAMESPACE}.request`
+const OTEL_REQ_QUERY = `${OTEL_REQ_NAMESPACE}.query`
+const OTEL_REQ_PARAM_NAMESPACE = `${OTEL_REQ_NAMESPACE}.param`
```

Comment:
Most properties were handled with standard Semantic Attributes (tm) - these cover the handful that weren't. I'm not *sure* what the otel convention is for custom attributes, so I made a best guess.

Reply:
there is a convention, LMK if you want me to track it down.

Reply:
Yeah, if you know where to look it would save me a lot of head-banging. I can take and run with docs if you have them.

Reply:
Here's the document I was thinking of: https://opentelemetry.io/docs/reference/specification/common/attribute-naming/
See the sections under "library authors" and "application developers".

After reading that, `boltzmann` like you have currently seems like a pretty reasonable namespace I think?
if we followed their examples for the HTTP property naming, the request namespace would become this:

```suggestion
const OTEL_REQ_NAMESPACE = `${OTEL_NAMESPACE}.http.request`
```